### PR TITLE
Add Anthropic docs

### PR DESCRIPTION
**Pull request by czyber**
The anthropic python sdk integration is documented and added to the integrations list

## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Add docs for Anthropic Integration for the python SDK.
https://github.com/getsentry/sentry-python/pull/2831
## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/python/integrations/anthropic/index.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+It takes a couple of moments for the data to appear in [sentry.io](https://sentry.io).
+
+## Behavior
+
+- The supported modules are currently `chat.messages.create` with `stream=True` and `stream=False`.
+
+- All exceptions leading to an `AnthropicError` are reported.
+
+- Sentry is configured not to consider LLM and tokenizer inputs/outputs as PII. If you want to include them, set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts despite `send_default_pii=True`, configure the integration with `include_prompts=False` like in the Options section.
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/anthropic/index.mdx:67]:
```suggestion
- Sentry is configured not to consider LLM and tokenizer inputs/outputs as PII. If you want to include them, set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts despite `send_default_pii=True`, configure the integration with `include_prompts=False`, as shown in [Options](#options).
```

**Review by vivianyentran — APPROVED** [review]
@sentrivana @antonpirker Can one of you take a look at this PR for a new python integration?

**Comment by antonpirker** [comment]
Yay, anthropic integration coming to Sentry. I will review the integration itself before reviewing the docs to make sure the info in the docs match what the integration does!

**Review by antonpirker — APPROVED** [review]
This looks good to me!

I will merge this when we release the Anthropic integration

**Review by antonpirker — APPROVED** [review]
looking good

**Pull request merged**